core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -558,8 +558,9 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |  --verbose, -v               Print additional debug output.
         |  --version,                  Print the version of current Spark.
         |
-        | Spark standalone with cluster deploy mode only:
-        |  --driver-cores NUM          Cores for driver (Default: 1).
+        | Cluster deploy mode only:
```

Comment:
Yes, I see the code used, although I am not familiar with mesos.
![image](https://user-images.githubusercontent.com/26266482/26924193-4e18d7a6-4c77-11e7-9496-681ea5a27eb6.png)